src/operator/tensor/ordering_op-inl.h

```diff
@@ -785,8 +813,12 @@ inline bool SortType(const nnvm::NodeAttrs& attrs,
   size_t out_size = out_attrs->size();
   CHECK_EQ(in_size, 1);
   CHECK_EQ(out_size, 2);
+#if MXNET_USE_INT64_TENSOR_SIZE == 1
+  CHECK(type_assign(&(*out_attrs)[1], mshadow::kInt64))
+#else
   CHECK(type_assign(&(*out_attrs)[1], mshadow::kInt32))
-          << "Failed to set the type of ret_indices to int32.";
+#endif
+      << "Failed to set the type of ret_indices to int32.";
```

Comment:
I think this message should be different when `MSHADOW_USE_INT64_TENSOR_SIZE` is on.

Reply:
I think its better if we logged "Failed to set the type of ret_indices"